### ISSUE TITLE
rather than inserting tridactyl iframe into pages insert pages as iframe into tridactyl page

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#1995]
Glacambre says `mozbrowser` lets us make iframes that don't know they're iframes.

We could therefore fix lots of the issues we have with pages loading by having Tridactyl outside of the page that's loading.

https://developer.mozilla.org/en-US/docs/Mozilla/Gecko/Chrome/API/Browser_API

We'd probably need to use the WebRequest API to redirect all requests through our own page (e.g. if someone opens a link from another application, or tries to use the Firefox URL bar).

**Comment by noctuid** [comment]
Would this address #1445?

**Comment by bovine3dom** [comment]
Yeah, that's the idea, but I don't think `<iframe mozbrowser>` exists for WebExtensions yet, we're waiting on this upstream bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1318532

**Comment by sanjay-kr-commit** [comment]
has there been any progress on this issue regarding extension not working while page is loading
